articles/gitops-mode-software.md

```diff
@@ -0,0 +1,34 @@
+# Manage software using GitOps
+
+This guide is a walkthrough on how to manage software using [GitOps](https://fleetdm.com/docs/configuration/yaml-files#gitops). In Fleet, there are three types of software: [Fleet-maintained apps](#fleet-maintained-apps), [App Store (VPP) apps](#app-store-vpp-apps), and [custom packages](#custom-packages).
+
+When using Gitops to manage Fleet, you can optionally put the UI in [GitOps mode](https://fleetdm.com/guides/gitops-mode). This prevents you from making changes in the UI that would be overridden by GitOps workflows.
+
+## Fleet-maintained apps
+
+Currently, managing [Fleet-maintained apps](https://fleetdm.com/guides/fleet-maintained-apps) is only supported using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (Gitops support coming soon. Currently, Gitops won't delete/modify Fleet-maintained apps configured in the UI).
```

Comment:
```suggestion
Currently, managing [Fleet-maintained apps](https://fleetdm.com/guides/fleet-maintained-apps) is only supported using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api).
```

@eugkuo we changed this right? As of the latest Fleet, we will delete Fleet-maintained apps added in the UI.